src/combinatorics/simple_graph/inc_matrix.lean

```diff
@@ -0,0 +1,409 @@
+/-
+Copyright (c) 2021 Gabriel Moise. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Gabriel Moise.
+-/
+
+import algebra.big_operators.basic
+import combinatorics.simple_graph.basic
+import data.fintype.basic
+import data.sym2
+import linear_algebra.matrix
+
+/-!
+# Incidence matrices
+
+This module defines the incidence matrix `inc_matrix` of an undirected graph `simple_graph`, and provides
```

Comment:
linter will shout about anything over 100 chars. they're all over the file like here and l96 and l240; i'd recommend going through them with the vscode margin turned on